.github/workflows/base.yaml

```diff
@@ -64,6 +81,7 @@ jobs:
           tags: ${{ steps.base_meta.outputs.tags }}
           labels: ${{ steps.base_meta.outputs.labels }}
           file: ${{ matrix.platform[0] }}/Dockerfile
+          outputs: type=image,name=${{ env.REGISTRY_IMAGE }},push-by-digest=true,name-canonical=true,push=false
```

Comment:
Not sure if this is needed, since this workflow apparently is just for warming up the S3 cache.